docs/user-guide/advanced/replicate_evaluations.md

```diff
@@ -2,179 +2,156 @@
 
 To produce the evaluation results presented [here](../../index.md#evaluation-results), you can run *eva* with the settings below.
 
-Make sure to replace `<task>` in the commands below with `bach`, `crc`, `mhist` or `patch_camelyon`.
+The `.yaml` config files for the different benchmark datasets can be found [here](https://github.com/kaiko-ai/eva/tree/main/configs/vision).
```

Comment:
i think we should add a comment here that the config files need to be downloaded

Reply:
done